src/components/ImageGeneration/imageResponseOptions.ts

```diff
@@ -0,0 +1,57 @@
+import type { Font } from 'node_modules/next/dist/compiled/@vercel/og/satori';
+
+export const imageResponseOptions = async ({
+  width,
+  height,
+  scalingFactor,
+}: {
+  width: number;
+  height: number;
+  scalingFactor: number;
+}) => {
+  return {
+    headers: {
+      'Cache-Control': `public, max-age=${60 * 60 * 1000 * 24}, immutable`,
+    },
+    width: width * scalingFactor,
+    height: height * scalingFactor,
+    fonts: await getFonts(), // Await properly within the async function
+  };
+};
+
+async function getFonts(): Promise<Font[]> {
+  // This is unfortunate but I can't figure out how to load local font files
+  // when deployed to vercel.
+  const [interRegular, interSemiBold, interBold] = await Promise.all([
+    fetch(`https://fonts.cdnfonts.com/s/19795/Inter-Regular.woff`).then((res) =>
+      res.arrayBuffer(),
+    ),
+    fetch(`https://fonts.cdnfonts.com/s/19795/Inter-SemiBold.woff`).then(
+      (res) => res.arrayBuffer(),
+    ),
+    fetch(`https://fonts.cdnfonts.com/s/19795/Inter-Bold.woff`).then((res) =>
+      res.arrayBuffer(),
+    ),
+  ]);
+
+  return [
+    {
```

Comment:
Because you are loading Inter font only, would be good to name the function accordingly for simplicity

Reply:
VERY WELL spotted! Nice one, I agree!